SmartEVSE-3/data/index.html

```diff
@@ -788,7 +813,12 @@ <h1 id="version" class="h5 mb-0 text-white-800">Version: </h1>
                       window.location.href = "/settings";
                     }
                     function postPWM(value) {
-                       $.post("/modem?pwm=" + value);
+                       $.post("/settings?override_pwm=" + value);
+                       event.preventDefault();
+                     }
+                     function postRequiredEVCCID() {
```

Comment:
I have worked a bit on adding vehicle profiles, where you can enter:
 - SoH, because in theory you can't reach 100% anymore if you compute it with the variables we have. Unless the car compensates in EVEnergyCapacity.Value, but it doesn't seem like mine does. 
 - If the car does not report EVEnergyCapacity.Value, you can define it
 - See and authorize EVCCID

This is a great start though, and we can extend on this.

Reply:
Sounds amazing! I suppose we could anticipate the device profiles feature, already start using the new API structure/logic within the changes of this pull request that would deliver the absolute basics, then open another pull later to introduce the 'real' feature? Not really sure whether the state stuff and other logic I hacked in would really be useful then, though.